src/main/java/seedu/address/logic/parser/MakeStatsCommandParser.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TYPE;
+
+import java.util.stream.Stream;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.MakeStatsCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new MakeStatsCommand
+ */
+public class MakeStatsCommandParser {
+
+    private static final String TYPE_GENDER = "g";
+    private static final String TYPE_AGE = "a";
+    /**
```

Comment:
We should leave a line here
```suggestion
    private static final String TYPE_AGE = "a";
    
    /**
```

Reply:
ok